robolectric/src/test/java/org/robolectric/shadows/ShadowAudioManagerTest.java

```diff
@@ -1538,6 +1540,23 @@ public void getDirectProfilesForAttributes_returnsEmptyListByDefault() {
     verify(callback).onAudioDevicesAdded(new AudioDeviceInfo[] {outputDevice});
   }
 
+  @Test
```

Comment:
This should be `@Config(minSdk = O)`.

Activity context is only available for O and above. See https://github.com/robolectric/robolectric/blob/master/shadows/framework/src/main/java/org/robolectric/shadows/ShadowActivity.java#L183